packages/widgets/bettinggame/src/components/TournamentWidget.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types'
+import EntityDetailApp from 'tocco-entity-detail/src/main'
+
+const TournamentWidget = props => {
+  const {tournamentKey, formBase} = props
+  return (
+    <EntityDetailApp
+      id={`detail_${formBase}_${tournamentKey}`}
+      entityName="Tournament"
```

Comment:
suggestion(blocking): The `entityName` should come from the input props

The `formBase` and `entityName` are both immutable configs form the widget config. Please also use `entityName` from the config instead of using hardcoded name here.